src/userSurvey.js

```diff
@@ -171,7 +196,6 @@ export async function setSurvey(options) {
     let lastSurveyRequest = await storageSpace.get("lastSurveyRequest");
     const surveyCompleted = await storageSpace.get("surveyCompleted");
     const surveyCancelled = await storageSpace.get("surveyCancelled");
-    lastSurveyRequest = await storageSpace.get("lastSurveyRequest");
```

Comment:
Don't we need this? Otherwise calling `setSurvey` will always pop a tab open.

Reply:
We're already setting the value of `lastSurveyRequest` from storage 3 lines above this.